sdcm/sct_config.py

```diff
@@ -1044,6 +1045,15 @@ def __init__(self):
         if self.get('instance_provision') == 'mixed':
             LOGGER.warning('Selected instance_provision type "MIXED" is not supported!')
 
+        urls = [self.get('new_scylla_repo', None), self.get('scylla_repo_m', None),
```

Comment:
please move this check into `verify_configuration` function. and into the constructor

Reply:
a new flag ? why ?